scripts/gulp/html.js

```diff
@@ -83,7 +87,9 @@ function compileMD0(srcFiles) {
                 let html = hbsInstance.compile(node.value)({});
                 html = prettyhtml(html)
                   .contents.replace(/class /g, '')
-                  .replace(/ class>/g, '>');
+                  .replace(/ class>/g, '>')
+                  .replace(/[\n\t\s\r]+&nbsp;[\n\t\s\r]+/g, '&nbsp;')
```

Comment:
I'm not super hip to regex, but if I'm not mistaken, this removes any char that creates space around `&nbsp`'s? If so, it's valid to have spaces around `&nbsp` so I'm not sure we want to remove them with no way to preserve them. https://codepen.io/mcoker/pen/yLjzQLO

Reply:
You are right, maybe I can leave this to @mattnolting because I have already created `removeWhiteSpaceForPrettier` helper which he can use before and after `&nbsp;` where he wants to remove all the spaces around it so it will be more specific and will not influence the whole app.

Reply:
@mcoker I removed this line and used `{{removeWhiteSpaceForPrettier}}` where needed. Ready for review.